### ISSUE TITLE
about Usage

### ISSUE THREAD

**Issue by lin520chong** [heshenghuan/LSTM-CRF#1]
when can you write the Usage?

**Comment by heshenghuan** [comment]
@lin520chong 
sorry, I'm not sure...
Because this project is unfinished.
It will takes at most 2 weeks to be finished, I think... 😅

**Comment by lin520chong** [comment]
@heshenghuan  waiting for you
I'm also doing some work about LSTM for NER recently.
Maybe we can share our ideas when you finish this project.

**Comment by heshenghuan** [comment]
@lin520chong 
sorry, I'm tortured by some Keras bugs recently, because I found that the same code will generate different result on CPU and GPU. That's really disappoint me, cause I cannot find any logical error on my code. So I decided to use tensorflow to implement a LSTM model. And it works well, I'm runing a test now.

And inspired by some LSTM Chinese word segmentation (check this: [kcws](https://github.com/koth/kcws)), I no longer use single LSTM layer with 'softmax' activation, but added a CRF at the end of activation layer. I think this might be helpful to do structured predition.
